autoPlay.user.js

```diff
@@ -252,6 +254,29 @@ function disableParticles() {
 	}
 }
 
+function doRemoveLowTier() {
+  var map = {
```

Comment:
Can we have a global "enum" array of upgrades? Would make way more sense than these ids